src/mempool_options.h

```diff
@@ -22,6 +25,7 @@ struct MemPoolOptions {
     CBlockPolicyEstimator* const estimator = nullptr;
     int check_ratio = 0;
     size_t max_size = DEFAULT_MAX_MEMPOOL_SIZE_MB * 1000000;
+    int64_t expiry = DEFAULT_MEMPOOL_EXPIRY;
```

Comment:
How about `std::chrono::seconds`? It seems that's how it's used/compared in our code anyway

Reply:
It could be `NodeClock::duration` with the reason that the value is compared against the value of `GetTime()`, which is deprecated in favour of (and an alias to) `NodeClock::now()`.